apps/fishing-map/features/analysis/Analysis.tsx

```diff
@@ -161,10 +164,32 @@ function Analysis() {
 
   const onAnalysisTypeClick = useCallback(
     (option: ChoiceOption) => {
+      if (option.id === 'beforeAfter') {
+        uaEvent({
+          category: 'Analysis',
+          action: `Click '${option.title}' in analysis mode`,
+          label: JSON.stringify({
+            regionName: analysisAreaName,
+            sourceNames: dataviews.flatMap(dataview => getSourcesSelectedInDataview(dataview).map(source => source.label))
+          })
+        })
+      }
+      if (option.id === 'periodComparison' && timeComparison) {
+        console.log(timeComparison)
+        uaEvent({
+          category: 'Analysis',
+          action: `Click '${option.title}' in analysis mode`,
+          label: JSON.stringify({
+            duration: timeComparison.duration + ' ' + timeComparison.durationType,
```

Comment:
would timeComparison have the updated value at this point?

Reply:
I don't know a reason to don't have it updated, this is fired when we change tabs